### ISSUE TITLE
Installer - Silent command-line switch: Add alternative switch which outputs no audio whatsoever during installation

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#11622]
### Is your feature request related to a problem? Please describe.

When running the main NVDA package with the "--install-silent" command-line switch, NVDA is installed onto the system with no manual user intervention required. However, the NV Access musical logo is still played, and a temporary copy of NVDA is briefly started until the installation process has completed. This has the potential to be a problem for multiple reasons:

1. It may be a fair assumption that due to the word "silent" as part of the switch, system admins may think that no output whatsoever is made as part of the installation process - so may not expect any audio to be heard whatsoever

2. In environments where NVDA is being installed onto potentially thousands of workstations as part of a single deployment, for machines that have their audio outputs enabled this has the potential to cause confusion and disruption - imagine a class of 50 students and 50 separate machines all installing at once!

Whilst the above are important to consider, it's also worth noting that in certain situations, it may also be useful to have audio indication as to whether NVDA has been installed. For example, if deploying NVDA to a machine over a network, it may not immediately be obvious to a totally blind user as to whether NVDA has been installed onto a particular machine - and so having a brief audio logo would help in determining this.

Taking all of the above into account, it would be really useful if an additional command-line switch (something along the lines of "--install-silent-no-audio") could be added so that users/administrators can deploy NVDA depending on their environment/usage needs.

**Comment by XLTechie** [comment]
I can see the use cases for this.

For ease of documentation and possibly ease of implementation, I would suggest a 
separate "--no-audio" or "--mute" option to perform this task, rather than a 
separate implementation of the silent install option.

Such a sub-option would have no effect while not in a silent installation mode, 
naturally.

**Comment by feerrenrut** [comment]
I think perhaps it's a good idea to go the other way. Make `--silent` totally silent (visually and audibly) and have an override for `--with-audio`

With the goal of making it as easy as possible for sys-admins (for whom the silent option is intend) to install NVDA without running into surprising behavior.

**Comment by leonardder** [comment]
I think the initial request in this was covered by #12322, i.e. --minimal now acts as such.
I agree that silent really should be sileent. CC @JulienCochuyt

**Comment by Qchristensen** [comment]
While some of the original intent has been addressed, it is still not possible to install without progress dialogs appearing.

I had an email request from a system administrator for help with this.

**Comment by macxcool** [comment]
--minimal only gets rid of the music at the start and end, it doesn't stop NVDA from 'speaking'. It still reads anything on the screen or on any dialogs that pop up. Our school board uses a system that installs software in the background. I've taken to muting the volume with the install script before the install and un-muting it afterwards. ;-)

**Comment by Adriani90** [comment]
I agree this one could be very useful for sysadmins. cc: @gerald-hartig

**Comment by Qchristensen** [comment]
This is an ongoing concern for system administrators.  The closest I can find is if you run Command Prompt as administrator and run the installer (or just run the installer as administrator) with:
--install-silent --minimal

It still reads when it pops up the progress bar.  Additionally mine beeped (because progress bar) although the video a system admin just sent me didn't.

I had a look to see if there was a way of muting the system volume, so we could at least offer that as a workaround - there are, but not built in - you'd need to have a third-party tool for it.  If anyone is interested, this page had a few suggestions: https://superuser.com/questions/42039/change-windows-sound-volume-via-the-command-line
